### PR TITLE
Add geocoding API key environment property.

### PR DESCRIPTION
**Pull request by alxdarksage**

### CODE REVIEW

**Review by DwayneJengSage — CHANGES_REQUESTED** [review]
You also need to add to the parameters in templates/bridgeserver2.yaml.

To check the syntax on your template, run `cfn-lint ./templates/*`

File: templates/bridgeserver2.yaml [hunk 1]
```diff
@@ -380,6 +380,9 @@ Resources:
         - Namespace: 'aws:elasticbeanstalk:application:environment'
           OptionName: synapse.oauth.client.secret
           Value: !Ref SynapseOAuthClientSecret
+        - Namespace: 'aws:elasticbeanstalk:application:environment'
+          OptionName: crc.geocode.api.key
+          Value: !Ref CrcGeoCodeApiKey
```
Comment (zaro0508) [inline templates/bridgeserver2.yaml:388]: you need to add `CrcGeoCodeApiKey` to the `Parameters:` section of this file

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: templates/bridgeserver2.yaml (hunk 1 above)
Comment (alxdarksage) [reply templates/bridgeserver2.yaml:388]: Added without echo to terminal.

**Comment by alxdarksage** [comment]
Passes cfn-lint.

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**